package/src/registry.h

```diff
@@ -1,90 +1,102 @@
-// Copyright (C) CERN, 2020
+// Copyright (C) 2018-2020 CERN
+// License Apache2 - see LICENCE file
+
+// Implementation of a class registry that is used
+// for all prmon monitoring classes
+//
+// Grateful acknowledgement to https://github.com/psalvaggio/cppregpattern
+// which inspired this implementation of the pattern
```

Comment:
I think this is great. Maybe we can also add an `doc/ACKNOWLEDGEMENTS.md` and collate the information there instead, which would be easier to list and identify.

Reply:
Also a good idea - I rearranged a little between `ACKNOWLEDGEMENTS.md` and `AUTHORS.md` as well

I did keep it in the root directory as it's not really documentation, right?